pkgs/servers/livekit/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec {
+  pname = "livekit";
+  version = "1.2.3";
+
+  src = fetchFromGitHub {
+    owner = "livekit";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-O+nmPbh5msecYrMcXGzh+JmNXKjHt+M66yMYZ7LRDjc=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.